InternetHospital.WebApi/Controllers/ModeratorController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Threading.Tasks;
+using InternetHospital.BusinessLogic.Interfaces;
+using InternetHospital.BusinessLogic.Models;
+using InternetHospital.DataAccess.Entities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+namespace InternetHospital.WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize(Policy = "Administrator")]
```

Comment:
You could use [Authorize(Roles = "Admin")] instead of create a policy with the same function